scripts/rook.sh

```diff
@@ -222,3 +238,6 @@ Available Commands:
 " >&2
 	;;
 esac
+
+# no major error occured, no need to call the logging function on exit
+trap - EXIT
```

Comment:
This is not needed anymore since you trap for `ERR` now.

Reply:
duh :man_facepalming: